testapi.pm

```diff
@@ -712,7 +714,9 @@ Run C<$command> via C<script_sudo> and then check by C<wait_serial> if its exit
 status is not zero.
 See C<wait_serial> for default timeout.
 
-I<The implementation is distribution specific and not always available.>
+I<The implementation is distribution specific and not always available. Note this
+is quite likely to fail with the default C<script_sudo> implementation, as it attempts
+to echo to the serial device without root permissions.>
```

Comment:
sure, sounds reasonable. I guess I was assuming you were just only ever using `script_run` as root, like us. :) I like your text, I'll adjust to use it and maybe add the same note in `script_run`.